.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,7 @@
 **Jira issue:**
-https://glia.atlassian.net/browse/MUIC-xxx
+https://glia.atlassian.net/browse/MOB-xxx
+
+**Release notes?:** (Feature, Ignore, Release notes, Migration guide)
```

Comment:
What do you think regarding adding checkboxes?

**Release notes:**
- [ ] Feature
- [ ] Ignore
- [ ] Release notes
- [ ] Migration guide